frontend/src/report/bar-chart/BarChartReport.svelte

```diff
@@ -6,7 +6,7 @@
 	import generateSpec from "./vegaSpec-bar";
 
 	$: currentReport = $reports[$report];
-	$: selectMetrics = currentReport.metrics;
+	$: selectMetrics = currentReport.metrics[0];
```

Comment:
small bug: if metrics is not fixed and I remove all metrics, when I make it fixed this breaks. When we switch a dimension to fixed we should check if it is empty first and add the first metric to it if it's empty.

Reply:
Woo, thx for pointing out the bug, the logic you mentioned makes sense to me, will fix it

Reply:
restore/add all values instead of the first value to the empty dimension before fixing it to increase the flexibility.